presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -27,22 +21,23 @@ services:
       - '10000:10000'
       - '50070:50070'
       - '50075:50075'
-  
-  presto-master-common:
+
+  presto-master:
     extends:
       service: java-8-base
     hostname: presto-master
     volumes:
       - ../../../../presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
     ports:
       - '8080:8080'
+    depends_on:
+       - hadoop-master
 
-  presto-worker-common:
-    extends:
-      service: java-8-base
-    command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh multinode-worker run
-
-  application-runner-common:
+  application-runner:
     extends:
       service: java-8-base
     hostname: application-runner
+    depends_on:
```

Comment:
It shouldn't depend on `presto-master`. Application runner is independent container to run the ad-hoc application, and shouldn't be run as a deamon.